homework_1/task2/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"code-cadets-2021/homework1/task2/progressivetax"
+	"github.com/pkg/errors"
+)
+
+func main() {
+
+	fmt.Println("Enter amount earned: ")
+
+	var enteredIncome float32
+	_, err := fmt.Scanf("%f", &enteredIncome)
+	if err != nil {
+		log.Fatal(
+			errors.WithMessage(err, "while entering income"))
+	}
+
+	var taxBracket = []progressivetax.Bracket{
+		{
+			TaxRate:   0,
+			Threshold: 0,
+		},
+		{
+			TaxRate:   10,
+			Threshold: 1000,
+		},
+		{
+			TaxRate:   20,
+			Threshold: 5000,
+		},
+		{
+			TaxRate:   30,
+			Threshold: 10000,
+		},
+	}
+
+	calculatedTax, err := progressivetax.CalculateProgressiveTax(enteredIncome, taxBracket)
+	if err != nil {
+		log.Fatal(
+			err,
+		)
```

Comment:
Could be just log.Fatal(err), you should split line if it goes beyond 80/100/120 characters (depending on what the team agrees on - we usually use 100 characters). This is nowhere near the limit.

Reply:
Understood, I saw it being used with longer errors and Golands code refraction didn't combine it so i thought it was good to leave it like that.